### PR TITLE
ScyllaDB deployment for kubernetes

### PR DESCRIPTION
**Pull request by smacker**
For now supports only one instance in cluster.
Tested on minikube.

### CODE REVIEW

**Comment by bzz** [comment]
Did not try it yet with kubectl.
\cc @rporres for a quick glance, in case of some spare cycles.

File: k8s/scylla-statefulset.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1beta2
```
Comment (rporres) [inline k8s/scylla-statefulset.yaml]: I think k8s 1.5 does not support this apiVersion

File: k8s/scylla-statefulset.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          initialDelaySeconds: 15
+          timeoutSeconds: 5
+        volumeMounts:
+          - name: scylla-data
+            mountPath: /var/lib/scylla
+          - name: scylla-ready-probe
+            mountPath: /opt/ready-probe.sh
+            subPath: ready-probe.sh
+      nodeSelector:
+        # add selecter here when node is ready
```
Comment (rporres) [inline k8s/scylla-statefulset.yaml]: `srcd.host/type: worker-applications`

Comment (rporres) [reply k8s/scylla-statefulset.yaml]: Or if you want something more restrictive, you can use `hostname: worker-4.k8s.pipeline.staging.srcd.host`

**Comment by smacker** [comment]
Thanks! Version changed, selector added.
I thought to create helm chart, but Alex said just such config is enough. We will migrate it to helm later.

**Comment by rporres** [comment]
If you're going to put the database in disk, most likely you want to tie your stateful set to a host rather than a set of hosts as there are three `srcd.host/type: worker-applications`. If not, it's ok

**Comment by smacker** [comment]
Yeah, thanks. I'll discuss it with Alex. Normally we would need persistent volume, but for testing just mounted directory should be fine.

Tried to deploy, it works:
```
smacker at Maxims-MacBook-Air in ~/srcd/gemini on scylla_db_k8s*
$ kubectl exec -it rafa-test-3698564540-pzn0r sh
/ # telnet scylladb 9042

�1
+Invalid or unsupported protocol version: 13Connection closed by foreign host
/ #
```

P.S. sorry for using your pod :-D

**Comment by bzz** [comment]
@rporres thank you for as usually thoughtful feedback!

@smacker nice! BTW what is `DCO_2` file for?

**Comment by smacker** [comment]
oh. sorry. Haven't noticed test commit. Removed.

**Comment by smacker** [comment]
@bzz is anything else required here or we can merge?

**Pull request merged**